moveit_core/version/CMakeLists.txt

```
@@ -1,3 +1,31 @@
+# Generate and install version.h
+
+if(NOT "$ENV{USER}" STREQUAL "buildfarm") # Don't define version postfix on buildfarm
+	set(MOVEIT_VERSION_EXTRA "devel" CACHE STRING "version string postfix")
```

```suggestion
    set(MOVEIT_VERSION_EXTRA "devel" CACHE STRING "version string postfix")
```
Looks like someone used tabs in this file instead of spaces. Just had to mention that.